src/shipit_static_analysis/shipit_static_analysis/clang.py

```diff
@@ -141,7 +145,7 @@ def run_clang_tidy(self, checks):
             cmd = [
                 self.binary,
                 # Show warnings in all in-project headers by default.
```

Comment:
Nit: This comment should probably be changed to something like `Only show warnings in the currently analyzed file`.